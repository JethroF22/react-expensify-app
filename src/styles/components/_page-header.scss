.page-header {
  background: $off-white;
  padding: $large-size;
  margin-bottom: $large-size;
}

.page-header__actions {
  margin-top: $medium-size;
}

.page-header__title {
  margin: 0;
  font-weight: 300;
  span {
    font-weight: 700;
  }
}
