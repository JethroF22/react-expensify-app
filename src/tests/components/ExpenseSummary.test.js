import React from "react";
import { shallow } from "enzyme";
import { ExpenseSummary } from "../../components/ExpensesSummary";

test("should render correctly with 1 expense", () => {
  const wrapper = shallow(<ExpenseSummary expenseCount={1} expensesTotal={124183} />);
  expect(wrapper).toMatchSnapshot();
});

test("should render correctly with multiple expensesu", () => {
  const wrapper = shallow(<ExpenseSummary expenseCount={14} expensesTotal={23874293} />);
  expect(wrapper).toMatchSnapshot();
});
