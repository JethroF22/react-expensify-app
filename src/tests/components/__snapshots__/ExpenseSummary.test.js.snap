// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly with 1 expense 1`] = `
<div>
  <h2>
    Viewing 
    1
     
    expense
     totalling 
    $1,241.83
  </h2>
</div>
`;

exports[`should render correctly with multiple expensesu 1`] = `
<div>
  <h2>
    Viewing 
    14
     
    expenses
     totalling 
    $238,742.93
  </h2>
</div>
`;
